### PR TITLE
Added issue token page

### PR DESCRIPTION
**Pull request by razvantomegea**

### CODE REVIEW

File: src/pages/IssueToken/IssueToken.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { IssueTokenForm } from './components';
+
+export const IssueToken = () => {
+  return (
```
Comment (MiroMargineanu) [inline src/pages/IssueToken/IssueToken.tsx:4]: This could be an implicit `return`. 👍

File: src/pages/IssueToken/components/IssueTokenForm.tsx [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+import { useIssueToken } from '../hooks';
+import { IssueTokenFieldsEnum } from '../types';
+
+export const IssueTokenForm = () => {
+  const { formik } = useIssueToken();
+
+  return (
+    <form
+      onSubmit={formik.handleSubmit}
+      noValidate
```
Comment (MiroMargineanu) [inline src/pages/IssueToken/components/IssueTokenForm.tsx:15]: Does this property exist on the default HTML5 `form`?

File: src/pages/IssueToken/hooks/useIssueToken.ts [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        await sendTransactions([transaction]);
+      } catch (err) {
+        //setErrors({ amount: err.message });
+      }
+
+      formik.resetForm();
+    }
+  });
+
+  return {
```
Comment (MiroMargineanu) [inline src/pages/IssueToken/hooks/useIssueToken.ts:96]: Is returning an object needed considering you only have one property?

File: src/pages/index.tsx [hunk 4]
```diff
@@ -3,6 +3,7 @@ export * from './SovereignTransfer';
 export * from './Disclaimer';
```
Comment (MiroMargineanu) [inline src/pages/index.tsx:3]: This should be an `index.ts` considering you're not rendering any `JSX`.

File: src/pages/IssueToken/components/IssueTokenForm.tsx (hunk 2 above)
Comment (razvantomegea) [reply src/pages/IssueToken/components/IssueTokenForm.tsx:15]: https://www.w3schools.com/tags/att_form_novalidate.asp

File: src/pages/index.tsx (hunk 4 above)
Comment (razvantomegea) [reply src/pages/index.tsx:3]: I am exporting ONLY TSX files, pages

**Review by arhtudormorar — APPROVED** [review]

File: src/pages/IssueToken/components/IssueTokenForm.tsx (hunk 2 above)
Comment (MiroMargineanu) [reply src/pages/IssueToken/components/IssueTokenForm.tsx:15]: Cool. 👍

File: src/pages/index.tsx (hunk 4 above)
Comment (MiroMargineanu) [reply src/pages/index.tsx:3]: Cool. 👍

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- [Added faucet and issue token](https://github.com/multiversx/mx-wallet-dapp/pull/39)
```
Comment (MiroMargineanu) [inline CHANGELOG.md:10]: Are these the same `Pull Request`?

File: src/localConstants/misc.ts [hunk 6]
```diff
@@ -3,3 +3,4 @@ export const WALLET_FILE = 'walletFile';
 export const MAX_ALLOWED_TRANSACTIONS_TO_SIGN = 5;
 export const CUSTOM_TOAST_DEFAULT_DURATION = 5000;
 export const SOVEREIGN_TRANSFER_GAS_LIMIT = 20_000_000;
+export const TOKEN_KEY: any = 'token';
```
Comment (MiroMargineanu) [inline src/localConstants/misc.ts:6]: This is still `any` although it should be a string, inferred.

**Review by MiroMargineanu — APPROVED** [review]

**Pull request merged**